browser/js/components/Comment/test.js

```diff
@@ -0,0 +1,151 @@
+'use strict'
+
+import React, {
+  Component,
+  PropTypes
+} from 'react'
+import RaisedButton from 'material-ui/RaisedButton'
+import FlatButton from 'material-ui/FlatButton'
+import FontIcon from 'material-ui/FontIcon'
+import { connect } from 'react-redux';
+import {
+  Card,
+  CardActions,
+  CardHeader,
+  CardText
+} from 'material-ui/Card'
+import {
+  RadioButton,
+  RadioButtonGroup
+} from 'material-ui/RadioButton'
+import Grade from '../../containers/Grade'
+const _ = require('lodash');
```

Comment:
@thejohnbackes It would be better to just import the function you need from lodash, not the whole lodash library. 👍